src/azure-cli-core/azure/cli/core/tests/test_util.py

```diff
@@ -184,5 +184,141 @@ def test_b64_to_hex_type(self):
         self.assertIsInstance(b64_to_hex(self.base64), str)
 
 
+class TestHandleException(unittest.TestCase):
+
+    @mock.patch('azure.cli.core.util.logger.error', autospec=True)
+    def test_handles_keyboardinterrupt(self, mock_logger_error):
+        from knack.util import CLIError
+
+        # create test KeyboardInterrupt Exception
```

Comment:
Note: it's nice to add comments, but these comments are redundant. Comments should be used to explain logics which are not straightforward otherwise it makes the code verbose. Try to make your code readable to avoid comments.